src/parser.rs

```diff
@@ -2139,16 +2159,41 @@ impl<'a> Parser<'a> {
         Ok(ObjectName(idents))
     }
 
+    /// Parse identifiers strictly i.e. don't parse keywords
```

Comment:
maybe calling this `parse_identifiers_non_keywords` would be a more specific name